TestVectorsAwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts

```
@@ -66,25 +66,23 @@ repositories {
 dependencies {
     implementation("org.dafny:DafnyRuntime:${dafnyVersion}")
     implementation("software.amazon.smithy.dafny:conversion:0.1")
-    implementation("software.amazon.cryptography:aws-cryptographic-material-providers:1.2.0-SNAPSHOT")
+    implementation("software.amazon.cryptography:aws-cryptographic-material-providers:1.2.0")
+    implementation("software.amazon.cryptography:StandardLibrary:1.0-SNAPSHOT")
+    implementation("software.amazon.cryptography:ComAmazonawsKms:1.0-SNAPSHOT")
+    implementation("software.amazon.cryptography:ComAmazonawsDynamodb:1.0-SNAPSHOT")
+    implementation("software.amazon.cryptography:AwsCryptographyPrimitives:1.0-SNAPSHOT")
```

When I first read this, I was confused. 
I think I have figured out what is going on.
Can you verify my suggested comment is correct?
Also, can you create a GHI or SIM and update the TODO?

```suggestion
    // TODO: Automate keeping MPL version here and in MPL the same
    implementation("software.amazon.cryptography:aws-cryptographic-material-providers:1.2.0")
    // The following "transitive" dependencies of the MPL, in Java, are distributed in MPL as a "Fat Jar"
    // But the "Fat Jar"  is not composed in CI, only during publication.
    // So we directly depend on the SNAPSHOT versions, 
    // via the `settings.gradle.kts` file. 
    implementation("software.amazon.cryptography:StandardLibrary:1.0-SNAPSHOT")
    implementation("software.amazon.cryptography:ComAmazonawsKms:1.0-SNAPSHOT")
    implementation("software.amazon.cryptography:ComAmazonawsDynamodb:1.0-SNAPSHOT")
    implementation("software.amazon.cryptography:AwsCryptographyPrimitives:1.0-SNAPSHOT")
```